CMakeLists.txt

```diff
@@ -1,41 +1,99 @@
 cmake_minimum_required(VERSION 3.16)
 set(OpenGL_GL_PREFERENCE GLVND)
 
+option(USE_VCPKG "Use vcpkg to handle some of the dependencies" OFF)
```

Comment:
Would it be possible to wrap the whole VCPKG initialization into a script in the [`cmake`](https://github.com/Sygmei/ObEngine/tree/master/cmake) directory ?